src/azul/service/hca_response_v5.py

```diff
@@ -252,20 +252,20 @@ def agg_values(function: Callable[[JSON], T], *path: str) -> List[T]:
             return list(map(function, values))
 
         return SummaryRepresentation(
-            projectCount=agg_value('projectCount', 'value'),
+            projectCount=agg_value('project', 'doc_count'),
             specimenCount=agg_value('specimenCount', 'value'),
             speciesCount=agg_value('speciesCount', 'value'),
-            fileCount=agg_value('fileCount', 'value'),
-            totalFileSize=agg_value('total_size', 'value'),
+            fileCount=agg_value('fileFormat', 'doc_count'),
+            totalFileSize=agg_value('totalFileSize', 'value'),
             donorCount=agg_value('donorCount', 'value'),
             labCount=agg_value('labCount', 'value'),
-            totalCellCount=agg_value('total_cell_count', 'value'),
+            totalCellCount=agg_value('totalCellCount', 'value'),
             organTypes=agg_values(OrganType.for_bucket,
                                   'organTypes', 'buckets'),
             fileTypeSummaries=agg_values(FileTypeSummary.for_bucket,
                                          'fileFormat', 'myTerms', 'buckets'),
```

Comment:
```suggestion
                                         'fileTypeSummaries', 'myTerms', 'buckets'),
```

Reply:
Not renaming this one because it corresponds to the name of a facet.